app/views/grades/_review_table.html.erb

```diff
@@ -4,7 +4,12 @@
   <%end%>
 	<tr><th style="text-align:center;">&nbsp;</th>
 		<% for i in 0...reviews.size%>
-			<th style="text-align:center;"><%= collabel %>&nbsp;<%=i+1%>&nbsp;</th>
+      <% user = User.find_by_id(AssignmentParticipant.find(ResponseMap.find(reviews[i].map_id).reviewer_id).user_id) %>
+        <% if(user.role.instructor? || user.role.ta?) %>
```

Comment:
The logic here is not correct. One TA can still be a student in another assignment.